scylla-cdc-replicator/src/replication_tests.rs

```
@@ -72,9 +73,10 @@ mod tests {
         )
         .await;
 
-        let schema = CDCRowSchema::new(&result.col_specs);
+        let schema = CDCRowSchema::new(result.column_specs());
 
-        for mut log in result.rows.unwrap_or_default() {
+        while let Some(log_res) = result.next().await {
+            let mut log = log_res.unwrap();
```

:wrench: Justify the `unwrap`. I'm quite sure we should return an error instead.

---

ah... ic... got blindside since previous use `unwrap` variant (`unwrap_or_default()` exactly), but now you mention it, this `unwrap_or_default()` will not cause panic...